apps/web/src/lib/components/ui/agent-editor/create-agent.svelte

```diff
@@ -0,0 +1,156 @@
+						<Input
+							id="title"
+							name="title"
+							bind:value={$formAgent.title}
+							placeholder="Agent's title"
+							class="border-input placeholder:text-muted-foreground focus-visible:ring-ring col-span-3 mt-1  block  h-9 w-full  rounded-md border bg-transparent px-3 text-sm shadow-sm ring-offset-0 transition-colors file:border-0 file:bg-transparent file:text-sm file:font-medium focus-visible:outline-none focus-visible:ring-1 focus-visible:ring-offset-0 disabled:cursor-not-allowed disabled:opacity-50"
+						/>
+					</div>
+					<div class=" mb-4 mt-4 flex cursor-pointer items-center gap-4 p-2">
+						<Toggle
+							id="publishedToggle"
+							name="published"
+							on:click={() => (published = !published)}
+							class=" h-7 scale-125 transform cursor-pointer border"
+						>
+							<Label
+								for="publishedToggle"
+								class="text-on-background cursor-pointer  font-semibold italic">published</Label
+							>
+						</Toggle>
+
+						<!-- Hidden input to capture the toggle state -->
+						<Input type="hidden" name="published" value={published} />
+					</div>
+				</div>
+				{#if $errors.title}
+					<p class="mb-2 text-red-500">{$errors.title}</p>
+				{/if}
+				<div class="mb-4">
+					<Label for="role" class="block text-sm font-medium ">Role</Label>
+					<Input
+						id="role"
+						name="role"
+						bind:value={$formAgent.role}
+						placeholder="Agent's role"
+						class="border-input placeholder:text-muted-foreground focus-visible:ring-ring col-span-3 mt-1  block  h-9 w-full  rounded-md border bg-transparent px-3 text-sm shadow-sm ring-offset-0 transition-colors file:border-0 file:bg-transparent file:text-sm file:font-medium focus-visible:outline-none focus-visible:ring-1 focus-visible:ring-offset-0 disabled:cursor-not-allowed disabled:opacity-50"
+					/>
+				</div>
+				{#if $errors.role}
+					<p class="mb-2 text-red-500">{$errors.role}</p>
+				{/if}
+				<div class="mb-4">
+					<Label for="description" class="block text-sm font-medium ">Description</Label>
+					<Textarea
+						id="description"
+						name="description"
+						bind:value={$formAgent.description}
+						placeholder="Describe the agent's purpose"
+						class="border-input placeholder:text-muted-foreground focus-visible:ring-ring
+					col-span-3 mt-1  block h-24 w-full resize-none rounded-md border bg-transparent px-3 text-sm shadow-sm ring-offset-0 transition-colors file:border-0 file:bg-transparent file:text-sm file:font-medium focus-visible:outline-none focus-visible:ring-1 focus-visible:ring-offset-0 disabled:cursor-not-allowed disabled:opacity-50 [&::-webkit-scrollbar]:hidden"
+					></Textarea>
+				</div>
+				{#if $errors.description}
+					<p class="mb-2 text-red-500">{$errors.description}</p>
+				{/if}
+				<div class="flex w-full gap-4">
+					<div class="mb-4 flex w-full flex-col">
+						<Label for="models" class="text-on-background mb-2 font-semibold">Models</Label>
+						<select
```

Comment:
Should this be a select component instead?
https://www.shadcn-svelte.com/docs/components/select